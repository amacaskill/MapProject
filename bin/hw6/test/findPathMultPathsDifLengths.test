# tests that the shortest path is chosen when multiple paths are possible 

CreateGraph g 

AddNode g c1 
AddNode g c2 
AddNode g c3 
AddNode g c4 
AddNode g c5

AddEdge g c1 c2 f
AddEdge g c1 c4 a 

AddEdge g c2 c3 d 
AddEdge g c2 c1 b
AddEdge g c2 c4 a 
AddEdge g c2 c5 c 

AddEdge g c4 c5 a 
AddEdge g c5 c3 a 

FindPath g c1 c3 
