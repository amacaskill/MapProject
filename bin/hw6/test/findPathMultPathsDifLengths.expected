# tests that the shortest path is chosen when multiple paths are possible 

created graph g 

added node c1 to g
added node c2 to g
added node c3 to g
added node c4 to g
added node c5 to g

added edge f from c1 to c2 in g
added edge a from c1 to c4 in g

added edge d from c2 to c3 in g
added edge b from c2 to c1 in g
added edge a from c2 to c4 in g
added edge c from c2 to c5 in g

added edge a from c4 to c5 in g
added edge a from c5 to c3 in g

path from c1 to c3:
c1 to c2 via f 
c2 to c3 via d